kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018,2020 Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
fix copyright line please

`# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates. `